test/Microsoft.Dnx.Testing.Framework/DnxSdk/Dth/DthMessageTypes.cs

```diff
@@ -9,8 +9,10 @@ public class DthMessageTypes
         public const string GetDiagnostics = nameof(GetDiagnostics);
         public const string ProtocolVersion = nameof(ProtocolVersion);
         public const string ChangeConfiguration = nameof(ChangeConfiguration);
+        public const string EnumerateProjectContexts = nameof(EnumerateProjectContexts);
 
         // responses
         public const string AllDiagnostics = nameof(AllDiagnostics);
+        public const string ProjectContexts = nameof(ProjectContexts);
```

Comment:
Did you just assign a constant to a constant just so you don't use the other constant? :laughing: